### DISCUSSION TITLE
Show total data in/out statistic for all client

### DISCUSSION THREAD

**Discussion by quyleanh** [wg-easy/wg-easy#184] [Ideas]
For now, there is only upload and download speed are shown. I would like to suggest a feature that shows total data in/out for all client.

Could you please consider to add this?

**Comment by Torontoboy51** [comment]
I second this idea.  It would be very useful info to have.  
Perhaps, show the data usage by daily/weekly/monthly etc on a client & system level.

**Reply by BETEP-ok** [reply]
Double second that.
Basic statistics is a must.
A very neat project screaming for few prettification touches.

**Comment by rootslv** [comment]
It would be nice if add  how many megabytes have been downloaded per user in statistics

**Reply by nevalashka** [reply]
It s already in admin panel. You can point mouse to download or upload icon and see how gb user used in all time.

**Reply by BETEP-ok** [reply]
Show me how to do that on mobile)))

**Comment by quyleanh** [comment]
Sorry @pheiduck for tagging you. Could you please consider implementing this one?

**Comment by pheiduck** [comment]
@quyleanh sure

**Reply by quyleanh** [reply]
Yay. I'm looking forward to seeing this one. Thank you in advance.

**Comment by suxscribe** [comment]
Hey! I've worked on the exact same idea. 

**Changes:**
- Moved RX and TX into separate columns and highlighted the values. And added Total RX / TX values below.
- Added controls block and RX / TX blocks wrapping on tablet and mobile as shown on the screenshots


**Here are the screenshots:**

<details>
  <summary>Desktop view</summary>

  ![desktop](https://github.com/wg-easy/wg-easy/assets/19854626/2c20d117-978c-4d7c-b5f0-8819f6eeff6c)

</details>

<details>
  <summary>Tablet view</summary>

  ![tablet](https://github.com/wg-easy/wg-easy/assets/19854626/b53faa10-ec21-4e6e-a855-f93d3e6fb02c)

</details>

<details>
  <summary>Mobile view</summary>

  ![mobile](https://github.com/wg-easy/wg-easy/assets/19854626/71559bee-aedb-49eb-bfc0-fdab04f0249c)

</details>

**Reply by peterlewis** [reply]
This is cool! I think we need to have a think about how we can surface shiny extras like this, in a way that doesn't detract from the simplicity of the default interface. As a MVP it could take the form of an optional variable such as `$UI_DETAILED_STATS` set to `true`? Thoughts? @pheiduck @WeeJeWel

**Reply by pheiduck** [reply]
Yep, very nice!
@suxscribe PR?

**Reply by BETEP-ok** [reply]
God bless you, comrade..)

is it possible to have some kind of time frames as well. 
to be able to see stats for last week/day/custom.
For every client.
And figure out who is the most traffic hungry.

Otherwise IF someone of the clients start misusing VPN for torrent downloading, or gets some botnet virus- I won’t be able to identify it remotely.
